devops/ansible-role-girder/tasks/main.yml

```diff
@@ -23,8 +23,10 @@
 - name: Install Girder
   pip:
     name: "{{ girder_package }}"
-    version: "{{ girder_version|default(omit) }}"
-    extra_args: "{{ girder_pip_extra_args|default(omit) }}"
+    version: "{{ girder_version if girder_version not in ['latest', 'release'] else omit }}"
+    extra_args: "{{ '--pre' if girder_version == 'latest' else omit }}"
+    state: "{{ 'latest' if girder_version in ['latest', 'release'] else omit }}"
+    editable: "{{ girder_development_mode }}"
```

Comment:
If girder_development_mode is True, then this functionally does `pip install -U --pre -e girder` which doesn't work -- editable mode expects a local directory, not a pypi package.